lib/widgets/game.dart

```diff
@@ -9,8 +9,8 @@ import 'package:jamaica/widgets/stars.dart';
 class Game extends StatefulWidget {
   final ContestSession contestSession;
   final UpdateCoins updateCoins;
-
-  const Game({Key key, this.contestSession, this.updateCoins})
+  final Function quizScore;
```

Comment:
Again think of verbs and action, not nouns here.
Also what is the typedef of this method. you cant just say Function. See how UpdateCoins is defined